src/main/java/org/jabref/gui/externalfiles/DownloadFullTextAction.java

```diff
@@ -112,9 +112,7 @@ private void downloadFullTexts(Map<BibEntry, Optional<URL>> downloads, BibDataba
                 Optional<Path> dir = databaseContext.getFirstExistingFileDir(Globals.prefs.getFilePreferences());
                 if (dir.isEmpty()) {
                     dialogService.showErrorDialogAndWait(Localization.lang("Directory not found"),
-                            Localization.lang("Main file directory not set!") + " "
-                                    + Localization.lang("Preferences")
-                                    + " -> " + Localization.lang("File"));
+                            (Localization.lang("Main file directory not set. Check the preferences (linked files) or the library properties.")));
```

Comment:
I think you have now one extra brace here.

Reply:
Great, is now resolved! 😃